spec/models/recipe_spec.rb

```diff
@@ -12,4 +12,17 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe "class methods" do
+    it "#cost" do
+    @tacos = Recipe.create!(name: "Tacos", complexity: 2, genre: "Mexican")
+
+    @beef = @tacos.ingredients.create!(name: "Ground Beef", cost: 7)
+    @salt = @tacos.ingredients.create!(name: "Salt", cost: 1)
+    @tortillas = @tacos.ingredients.create!(name: "tortillas", cost: 3)
+    @salsa = @tacos.ingredients.create!(name: "salsa", cost: 2)
+    @s_cream = @tacos.ingredients.create!(name: "Sour Cream", cost: 1)
+      expect(@tacos.cost).to eq(14)
+    end
```

Comment:
```ruby
def total_cost
  14
end
```

Does your assertion disprove that this is what your method looks like? Probably not. My suggestion, for instance methods, is to test more than 1 instance object so that way your code proves that the value isn't hard-coded and is dynamic.

```ruby
expect(@tacos.total_cost).to eq(14)
expect(@spaghett.total_cost).to eq(12) # create another recipe
```